tests/foreman/cli/test_satellitesync.py

```diff
@@ -56,12 +57,54 @@ def config_export_import_settings():
     """Check settings and set download policy for export.  Reset to original state after import"""
     download_policy_value = Settings.info({'name': 'default_download_policy'})['value']
     rh_download_policy_value = Settings.info({'name': 'default_redhat_download_policy'})['value']
-    disconnected_mode_value = Settings.info({'name': 'content_disconnected'})['value']
+    subs_conn_enabled_value = Settings.info({'name': 'subscription_connection_enabled'})['value']
     Settings.set({'name': 'default_redhat_download_policy', 'value': 'immediate'})
     yield
     Settings.set({'name': 'default_download_policy', 'value': download_policy_value})
     Settings.set({'name': 'default_redhat_download_policy', 'value': rh_download_policy_value})
-    Settings.set({'name': 'content_disconnected', 'value': disconnected_mode_value})
+    Settings.set({'name': 'subscription_connection_enabled', 'value': subs_conn_enabled_value})
+
+
+@pytest.fixture(scope='function')
+def export_import_cleanup_function(target_sat, function_org):
+    """Deletes export/import dirs of function org"""
+    yield
+    # Deletes directories created for export/import test
+    assert target_sat.execute(f'rm -rf {EXPORT_DIR}/{function_org.name}').stdout == ''
+    assert target_sat.execute(f'rm -rf {IMPORT_DIR}/{function_org.name}').stdout == ''
```

Comment:
In any case, `rm -rf` will work anyway and won't return anything, if directories exist or not, so I wouldn't assert here, and I would suggest deleting both dirs at once.
```suggestion
    target_sat.execute(f'rm -rf {EXPORT_DIR}/{function_org.name} {IMPORT_DIR}/{function_org.name}')
```

Reply:
You're right, the assert is not necessary here. I have removed it. But I would like to keep it a "two liner"